Voter.md

```diff
@@ -0,0 +1,85 @@
+# Voter SF b3
+
+Voters are Symfony's most powerful way of managing permissions. They allow you to centralize all permission logic, then reuse them in many places.
+
+Create file `/src/Security/YourBestVoter.php`.
+
+In this file you can create the class like that:
+```
+class YourBestVoter extends Voter {
+
+
+}
+```
+
+You will need to import to methods: supports and voteOnAttribute
+```
+protected function supports(string $attribute, $subject): bool
+{
+	// $attribute contains your parameter send by is_granted method
+	// only vote on `Post` objects
+	if (!$subject instanceof Post) {
+		return false;
+	}
+
+	return true;
+}
+
+protected function voteOnAttribute(string $attribute, $subject, TokenInterface $token): bool
+{
+	$user = $token->getUser();
+
+	// the user must be logged in; if not, deny access
+	if (!$user instanceof User) {
+		return false;
+	}
+	
+	if($subject->getUser() !== $user)
+		return false;
+	return true;
+}
+
+```
+
+## Explanation
+
+The methods `supports` is used for validate the request. If the return is false, the return code will be 401 Access denied. 
```

Comment:
Elaboration on the case of possibly multiple voters would be great. There is not always only one voter.
Further info : https://symfony.com/doc/5.4/security/voters.html#changing-the-access-decision-strategy